cli/internal/helm/README.md

```diff
@@ -0,0 +1,17 @@
+# cilium
+- Collect values using `helm get values`, merge with values from chart and apply. Needs to be done to pick up any newly introduced values.
+- `--reuse-values` is turned off.
```

Comment:
Could you add some context to this readme?
I imagine this to be a bit confusing if not read in the context of this PR